examples/plot_from_raw_to_epochs_to_evoked.py

```diff
@@ -49,7 +49,9 @@
                     baseline=(None, 0), reject=dict(eeg=80e-6, eog=150e-6),
                     preload=True)
 
-epochs.plot()
+# Two ways of viewing and rejecting the epochs.
+epochs.plot(block=True)
+epochs.plot(trellis=False, block=True, fig_title='Epochs')
```

Comment:
we could change the pick_types here: https://github.com/mne-tools/mne-python/pull/2154/files#diff-cf8fa71e8e2648014d98083b2a3cfd99R45